From 7eebd46b1a1332ca83b41805e138692a9c343e36 Mon Sep 17 00:00:00 2001
Message-Id: <RFC-patch-v3-3.9-7eebd46b1a1-20210806T173953Z-avarab@gmail.com>
In-Reply-To: <RFC-cover-v3-0.9-00000000000-20210806T173953Z-avarab@gmail.com>
References: <20210805230321.532218-1-mathstuf@gmail.com>
	<RFC-cover-v3-0.9-00000000000-20210806T173953Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 5 Aug 2021 15:35:42 +0200
Subject: [RFCPATCH v3 3/9] string-list API users: change confused remote.c
 user
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change a user of the string-list API in builtin/remote.c which
initialized with STRING_LIST_INIT_NODUP, and then right away did the
equivalent of having initialized with STRING_LIST_INIT_DUP by setting
the "strdup_strings" member to "1".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/remote.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/builtin/remote.c b/builtin/remote.c
index 7f88e6ce9de..b187b429516 100644
--- a/builtin/remote.c
+++ b/builtin/remote.c
@@ -1179,10 +1179,9 @@ static int get_one_entry(struct remote *remote, void *priv)
 
 static int show_all(void)
 {
-	struct string_list list = STRING_LIST_INIT_NODUP;
+	struct string_list list = STRING_LIST_INIT_DUP;
 	int result;
 
-	list.strdup_strings = 1;
 	result = for_each_remote(get_one_entry, &list);
 
 	if (!result) {
-- 
2.33.0.rc0.646.g806d90da684

